middleware/kubernetes/setup.go

```diff
@@ -196,7 +229,33 @@ func kubernetesParse(c *caddy.Controller) (*Kubernetes, error) {
 	return nil, errors.New("kubernetes setup called without keyword 'kubernetes' in Corefile")
 }
 
+func getSearchPathFromResolvConf(filename string) ([]string, error) {
```

Comment:
Yeah - i'm not handling domain and search path correctly anyways.  I was appending the two.  Apparently the domain is only used if search is not specified